src/app/item-popup/ItemPopup.m.scss

```diff
@@ -7,59 +7,54 @@
 @use '../variables.scss' as *;
 
 .arrow {
-  width: 10px;
-  height: 10px;
+  width: $theme-tooltip-arrow-size;
+  height: $theme-tooltip-arrow-size;
   border-style: solid;
   position: absolute;
-  border-color: black;
-  z-index: -1;
+  border-width: $theme-tooltip-arrow-size;
+  border-color: transparent;
 }
 
 .desktopPopupRoot {
   pointer-events: none;
-  .arrow {
-    border-color: #161626;
-  }
```

Comment:
We probably want the arrow's colour to match the actions background colour.

![image](https://github.com/DestinyItemManager/DIM/assets/17512262/4e0b67fe-8b1a-4e76-8b9b-32b166cbb8f5)